packages/react-server-test-pages/middleware/RequestToPort.js

```diff
@@ -1,10 +1,12 @@
 import {ReactServerAgent} from "react-server";
 export default class RequestToPortMiddleware {
 	handleRoute(next){
-		ReactServerAgent.plugRequest(req => {
-			// TODO: Get port dynamically?
-			req.urlPrefix('localhost:3000');
-		});
+		if (typeof window === "undefined"){
+			ReactServerAgent.plugRequest(req => {
+				// TODO: Get port dynamically?
+				req.urlPrefix('localhost:3000');
+			});
+		}
```

Comment:
This just disables the prefix in the browser since it causes cross origin errors.